README.md

```diff
@@ -1,92 +1,150 @@
+        - Copy `Registry\Desktop\Plugin\registry.dll` to `C:\Program Files (x86)\NSIS\Plugins\x86-unicode\registry.dll`
+
+Delete `pkg-config` files to prevent conflicts with Strawberry's own `pkg-config`:
+- Delete `C:\strawberry\perl\bin\pkg-config`
+- Delete `C:\strawberry\perl\bin\pkg-config.bat`
+
+Windows Settings | System | About | Advanced system settings | Tab Advanced | Environment Variables:
+- System variables | Path | Edit:
+    - Delete: `C:\Strawberry\c\bin` (To prevent conflicts with other utilities)
+    - Add: `C:\Program Files\Git\bin` (This is for `sed` and `patch` utilities)
+    - Add: `C:\Users\user\AppData\Local\bin\NASM`
+    - Add: `C:\data\projects\strawberry\msvc_\downloads\win_flex_bison-<VERSION>`
+    - Add: `C:\Program Files (x86)\NSIS`
+    - Make sure no MinGW-W64 (gcc, g++, etc) installation is in PATH, as this can cause those to be picked up as compiler instead of MSVC.
+
+### Meson installation
+
+Building with Meson 1.6.0 installation results in build issues
+[meson: error: unrecognized arguments:](https://github.com/strawberrymusicplayer/strawberry-msvc/issues/6).
+
+A temporary solution is to remove Meson via Windows Settings | `Apps or remove programs`.
+Then install Meson via Python PIP system wide. Start | `x64 Native Tools Command Prompt for VS 2022` and type:
+
+```
+pip install meson
+```
+
+## Optional: Prebuilt binary dependencies
+
+Prebuilt MSVC binaries can be optionally used to speed-up the build process. When this step is skipped, all libraries and dependencies are build from source.
+
+To use prebuilds, download the following `tar.xz` files from Github [strawberry-msvc-dependencies/releases](https://github.com/strawberrymusicplayer/strawberry-msvc-dependencies/releases):
+- For debug, extract `strawberry-msvc-x86_64-debug.tar.xz` to `C:\strawberry_msvc_x86_64_debug`
+- For release, extract `strawberry-msvc-x86_64-release.tar.xz` to `C:\strawberry_msvc_x86_64_release`
+
+## Build Strawberry from source including dependencies
+
+- Set one debug or release in `Path` environment variable:
+    - For debug: `C:\strawberry_msvc_x86_64_debug\bin`, or:
+    - For release: `C:\strawberry_msvc_x86_64_release\bin`
+- Start | `x64 Native Tools Command Prompt for VS 2022`:
+```
+cd C:\data\projects\strawberry\strawberry-msvc
+```
+
+For debug build:
+```
+.\build.bat
+```
+
+For release build:
+```
+.\build.bat release
+```
+
+Strawberry and Windows installer executables are generated in:
+- For debug: `C:\data\projects\strawberry\msvc_\build_debug\strawberry\build\`
+- For release: `C:\data\projects\strawberry\msvc_\build_release\strawberry\build\`
+
+## Debug Strawberry with Visual Studio
+
+Debug build:
+- Make sure path `C:\strawberry_msvc_x86_64_debug\bin` is set in environment variables
+- Start | Visual Studio 2022
+- Open a local folder `C:\data\projects\strawberry\msvc_\downloads\strawberry`
```

Comment:
Don't use this directory as it should be seen as a temporary directory used by the bat files, use: ``C:\data\projects\strawberry\strawberry``.

Reply:
I assume your idea is to add a clone of the Strawberry source repository in section `Windows setup`? I don't have it on my disk after running `download.bat` and `build.bat`:

```diff
mkdir C:\data\projects\strawberry
cd C:\data\projects\strawberry
+git clone git@github.com:strawberrymusicplayer/strawberry.git
git clone https://github.com/strawberrymusicplayer/strawberry-msvc.git
```